File: Review-04/Example-4/src/main/java/PlayTicTacToe.java
import java.io.IOException;

/**
 * The wrapper for main function.
 */
public class PlayTicTacToe
{
    public static void main(String[] args)
        throws IOException
    {
        Player tom = new Player("Thomas");
        Player jay = new Player("Jay");

        Game game = new Game(tom, jay);

        //while(!(game.getBoard().isFull())) {
        while (game.isNotOver()) {
            game.playRound();
        }

        System.out.println(game.getBoard());

        if (game.endedWithWin()) {
            System.out.printf("Congratulations %s!%n", game.getWinner());
        }
    }
}

File: Review-04-Unit-Testing/Example-6/play_tictactoe.py
#! /usr/bin/env python3

from examples import *


def main():

    tom = Player("Thomas")
    jay = Player("Jay")

    game = Game(tom, jay)

    # while(!(game.getBoard().isFull())) {
    while game.isNotOver():
        game.playRound()

    print(game.getBoard())

    if game.endedWithWin():
        print("Congratulations {}!".format(game.getWinner()))


if __name__ == "__main__":
    main()

    # @todo add exception handling
